### DISCUSSION TITLE
I can't reapply for the Github Education Pack

### DISCUSSION THREAD

**Discussion by enderatas** [community/community#72818] [GitHub Education]
### Hi everyone,

My Github Education Pack application has been rejected. I realized some mistakes I made and corrected them. I was going to reapply, but as you can see, the 'continue' button below is not clickable. What should I do? [attachment](https://ibb.co/R2xdDKC)

**Comment by mamoon-rasheed** [comment]
have you tried refreshing the page and trying again?
when you apply it will take the github to verify your profile
so i think it will be fixed

**Reply by enderatas** [reply]
I tried refreshing the page and logging out and logging in again. But I didn't get any results.

**Reply by APTEM591** [reply]
@enderatas did you solved it?
